scripts/src/owners/owners_file.py

```diff
@@ -61,7 +65,28 @@ def get_chart(owner_data):
     return chart
 
 
-def get_web_catalog_only(owner_data):
+def get_web_catalog_only(owner_data, raise_if_missing=False):
+    """Check the delivery method set in the OWNERS file data
+
+    Args:
+        owner_data (dict): Content of the OWNERS file. Typically this is the return value of the
+            get_owner_data or get_owner_data_from_file function.
+        raise_if_missing (bool, optional): Wether to raise an Exception if the delivery method is
```

Comment:
NIT: can be fixed in a future PR if you want.

```suggestion
        raise_if_missing (bool, optional): Whether to raise an Exception if the delivery method is
```